SkietbaanBE/SkietbaanBE/Properties/launchSettings.json

```diff
@@ -23,8 +23,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:63475/"
+      "applicationUrl": "http://localhost:63478/"
```

Comment:
No localhost vars 
Environment variables files are not supposed to be committed 
Empty line EOF